### PR TITLE
Create mishba-ai.mdx

### PR DESCRIPTION
**Pull request by mishba-ai**
## Fixes Issue

Closes #<issue_number>

### CODE REVIEW

File: contribution/mishba-ai.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Hi I'm misba bagban.I am currently a 1st year Computer engineering student persuing B-Tech.
+I love to contributing to OpenSource and getting involved in awesome tech communities.
+
+## Skills
+
+<Skills>
+  <Skill
+    name="Html"
+    icon={/* svg icon for Html */}
```
Comment (priyankarpal) [inline contribution/mishba-ai.mdx:17]: add svg icon

**Review by priyankarpal — CHANGES_REQUESTED** [review]
You can see the example profile here. https://github.com/pbclife/gitopener.vercel.app/blob/main/contribution/priyankarpal.mdx?plain=1

- You can add your skills
- projects
- etc

File: contribution/mishba-ai.mdx (hunk 1 above)
Comment (mishba-ai) [reply contribution/mishba-ai.mdx]: done

File: contribution/mishba-ai.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+   icon={
+   <svg xmlns="http://www.w3.org/2000/svg" class="icon icon-tabler icon-tabler-brand-html5" width="24" height="24" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round">
+   <title>html</title>
+   <path stroke="none" d="M0 0h24v24H0z" fill="none"></path>
+   <path d="M20 4l-2 14.5l-6 2l-6 -2l-2 -14.5z"></path>
+   <path d="M15.5 8h-7l.5 4h6l-.5 3.5l-2.5 .75l-2.5 -.75l-.1 -.5"></path>
+</svg>
+}
+  >
+   Html
```
Comment (priyankarpal) [inline contribution/mishba-ai.mdx:26]: Everything looks good. But could you add your experience such as `1-year experience with HTML` 

``` suggestion
1-year experience with HTML
```

File: contribution/mishba-ai.mdx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+ icon={ 
+ <svg xmlns="http://www.w3.org/2000/svg" class="icon icon-tabler icon-tabler-brand-css3" width="24" height="24" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round">
+ <title>CSS</title>
+   <path stroke="none" d="M0 0h24v24H0z" fill="none"></path>
+   <path d="M20 4l-2 14.5l-6 2l-6 -2l-2 -14.5z"></path>
+   <path d="M8.5 8h7l-4.5 4h4l-.5 3.5l-2.5 .75l-2.5 -.75l-.1 -.5"></path>
+</svg>
+}
+  >
+    CSS
```
Comment (priyankarpal) [inline contribution/mishba-ai.mdx:39]: same here also

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: contribution/mishba-ai.mdx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  <Skill
+    name="html"
+   icon={
+   <svg xmlns="http://www.w3.org/2000/svg" class="icon icon-tabler icon-tabler-brand-html5" width="24" height="24" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round">
+   <title>html</title>
+   <path stroke="none" d="M0 0h24v24H0z" fill="none"></path>
+   <path d="M20 4l-2 14.5l-6 2l-6 -2l-2 -14.5z"></path>
+   <path d="M15.5 8h-7l.5 4h6l-.5 3.5l-2.5 .75l-2.5 -.75l-.1 -.5"></path>
+</svg>
+level="Intermediate"
```
Comment (priyankarpal) [inline contribution/mishba-ai.mdx:27]: Remove this from here.

```suggestion

```

File: contribution/mishba-ai.mdx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+# Introduction
+
+Hi I'm misba bagban.I am currently a 1st year Computer engineering student persuing B-Tech.
+I love to contributing to OpenSource and getting involved in awesome tech communities.
+
+## Skills
+
+<Skills>
+  <Skill
+    name="html"
```
Comment (priyankarpal) [inline contribution/mishba-ai.mdx:19]: add this here. after `name`

``` add
level="Intermediate"
```

File: contribution/mishba-ai.mdx [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+  <Skill
+    name="CSS"
+ icon={ 
+ <svg xmlns="http://www.w3.org/2000/svg" class="icon icon-tabler icon-tabler-brand-css3" width="24" height="24" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round">
+ <title>CSS</title>
+   <path stroke="none" d="M0 0h24v24H0z" fill="none"></path>
+   <path d="M20 4l-2 14.5l-6 2l-6 -2l-2 -14.5z"></path>
+   <path d="M8.5 8h7l-4.5 4h4l-.5 3.5l-2.5 .75l-2.5 -.75l-.1 -.5"></path>
+</svg>
+level="Intermediate"
```
Comment (priyankarpal) [inline contribution/mishba-ai.mdx:41]: Also remove from here.
```suggestion

```

File: contribution/mishba-ai.mdx [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+   <path d="M20 4l-2 14.5l-6 2l-6 -2l-2 -14.5z"></path>
+   <path d="M15.5 8h-7l.5 4h6l-.5 3.5l-2.5 .75l-2.5 -.75l-.1 -.5"></path>
+</svg>
+level="Intermediate"
+}
+  >
+   Html
+  </Skill>
+  <Skill
+    name="CSS"
```
Comment (priyankarpal) [inline contribution/mishba-ai.mdx:33]: Add after `name`

```add
    level="Intermediate"
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — DISMISSED** [review]
Looks Good 🎉 You can customise your profile later.

**Review by sboy99 — APPROVED** [review]
LGTM 👌

**Pull request merged**